solutions/swb-reference/src/environment/sagemakerNotebook/sagemakerNotebook.cfn.yaml

```diff
@@ -70,6 +70,13 @@ Resources:
           FromPort: 443
           ToPort: 443
           CidrIp: !Ref AccessFromCIDRBlock
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W5
+            reason: 'At this point, this is by design. Customers should only be using this with test data.'
```

Comment:
This reason was for Public Preview. You should triage when you come back to this as we are aiming to not have it be used only on test data.